test/responders/guys_test.exs

```diff
@@ -0,0 +1,24 @@
+defmodule Elix.Responders.GuysTest do
+  use Hedwig.RobotCase
+
+  @tag start_robot: true, name: "elix", responders: [{Elix.Responders.Guys, []}]
+  test "hey guys", %{adapter: adapter, msg: msg} do
+    send adapter, {:message, %{msg | text: "hey guys"}}
+    assert_receive {:message, %{text: text}}
+    assert String.contains?(text, "Terms like *guys* can feel exclusionary. Did you mean **")
+    assert String.ends_with?(text, "**?")
+  end
+
+  @tag start_robot: true, name: "elix", responders: [{Elix.Responders.Guys, []}]
+  test "guys at beginning", %{adapter: adapter, msg: msg} do
+    send adapter, {:message, %{msg | text: "Guys, what do you think about this?"}}
+    assert_receive {:message, %{text: text}}
+    assert String.contains?(text, "*guys* can feel exclusionary.")
+  end
+
+  @tag start_robot: true, name: "elix", responders: [{Elix.Responders.Guys, []}]
+  test "guyser is ignored", %{adapter: adapter, msg: msg} do
+    send adapter, {:message, %{msg | text: "guyser"}}
+    refute_receive {:message}
+  end
```

Comment:
It looks like "guyser" actually does trigger a response. The reason I think `refute_receive` still passes is because it's pattern-matching on a one-element tuple of an atom, when the message received is a two-element tuple of an atom and map. Changing it to this:

```elixir
refute_receive {:message, %{}}
```

properly makes the spec fail. 

<img width="766" alt="screen shot 2016-10-08 at 9 01 32 am" src="https://cloud.githubusercontent.com/assets/664341/19213287/d3a935a2-8d35-11e6-966b-ce4e22ba9b4f.png">

And adding word boundaries around `guys` appears to fix it: `~r/\bguys\b/i`